website/static/js/logTextParser.js

```diff
@@ -220,25 +220,37 @@ var LogPieces = {
         view: function (ctrl, logObject) {
             var contributors = logObject.attributes.params.contributors;
             if(paramIsReturned(contributors, logObject)) {
-                return m('span', contributors.map(function(item, index, arr){
-                    var comma = ' ';
-                    if(index !== arr.length - 1){
-                        comma = ', ';
-                    }
-                    if(index === arr.length-2){
-                        comma = ' and ';
-                    }
+                return m('span', (function (){
+                    var contribList = [];
+                    var numContribShown = 15;
```

Comment:
Don't put magic numbers inside code; we'll want to make that something easy to change, and easy to find that it needs changing. In this case, probably at the top of the file under the imports. Make a note where it's used where it's defined.